src/main/java/org/spongepowered/common/event/tracking/PhaseTracker.java

```diff
@@ -84,25 +85,19 @@
 import org.spongepowered.common.event.tracking.phase.tick.NeighborNotificationContext;
 import org.spongepowered.common.event.tracking.phase.tick.TickPhase;
 import org.spongepowered.common.mixin.accessor.world.server.ServerWorldAccessor;
-import org.spongepowered.common.registry.type.event.SpawnTypeRegistryModule;
+import org.spongepowered.common.registry.builtin.supplier.SpawnTypeSupplier;
```

Comment:
Do not use suppliers in common code. Simple do SpawnTypes.BLAH.get()

Reply:
Can't if the spawn type is an internal one. It's not exposed to the API for the reason that it's internal use only but still detectable for plugins to interpret for their own uses.